.github/workflows/publish-gem.yml

```diff
@@ -0,0 +1,23 @@
+name: Publish Gem
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.7.3'
```

Comment:
Is there any reason not to release on a later version? At least worthwhile to use 2.7.8 to get the bug and security fixes.

Reply:
👋 @parkr do you think if we can move to ruby 3.0 for this project? yeh we can move to 2.7.8

Reply:
To add seems the same problem with CI can be seen with other repos (regarding Ruby version):
https://github.com/pages-themes/midnight/actions/runs/7391072383/job/20107215490?pr=50

Reply:
I think we should go to 3.2 at least, but there might be bugs to fix along the way.